test/integration/utils.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+
+import { BrowserWindow } from 'electron';
+
+const UI_DIR = path.join(__dirname, '..', '..', 'lib', 'ui');
+const BROWSER_CHROME_URL = fileUrl(path.join(UI_DIR, 'browser-modern', 'browser.html'));
+
+function fileUrl(str) {
+  let pathName = path.resolve(str).replace(/\\/g, '/');
+
+  // Windows drive letter must be prefixed with a slash.
+  if (pathName[0] !== '/') {
+    pathName = `/${pathName}`;
+  }
+
+  return encodeURI(`file://${pathName}`);
+}
+
+export function openBrowserWindow() {
```

Comment:
Why did this function have to be written? It's very similar to the one in the main process code. Unless I'm again missing something, should generalize some things into smaller utility methods so we don't write the same or similar code twice.